mm-bot/Makefile

```
@@ -8,13 +8,18 @@ database = mm-bot
 # Application Commands
 run:
 	@echo "Running application..."
-	@. venv/bin/activate && python3 main.py
+	@. venv/bin/activate && python3 src/main.py
 
 create_env:
 	@echo "Creating environment variables files .env",
 	@cp .env.example .env
 	@echo "Environment variables files created successfully!"
 
+create_venv:
```

```suggestion
create_python_venv:
```
what do you think? this is probably more descriptive to avoid any misunderstanding